crates/trigger/src/cli.rs

```diff
@@ -309,56 +300,17 @@ impl<T: Trigger> TriggerAppBuilder<T> {
     pub async fn build(
         &mut self,
         app: App,
-        options: TriggerAppOptions<'_>,
-    ) -> anyhow::Result<TriggerApp<T>> {
+        common_options: CommonTriggerOptions,
+        options: B::Options,
+    ) -> anyhow::Result<TriggerApp<T, B::Factors>> {
         let mut core_engine_builder = {
             self.trigger.update_core_config(&mut self.engine_config)?;
 
             spin_core::Engine::builder(&self.engine_config)?
         };
         self.trigger.add_to_linker(core_engine_builder.linker())?;
 
-        let runtime_config_path = options.runtime_config_file;
```

Comment:
All of this logic has been moved to the `RuntimeFactorsBuilder` implementation in `spin-cli`.